gourmet/shopEditor.py

```diff
@@ -1,4 +1,5 @@
-import gtk, gobject, backends.db, re, pickle
+from gi.repository import GObject, Gtk
+import backends.db, re, pickle
```

Comment:
I'm guessing `backends.db` needs to be a relative import (`from .backends import db` or similar)

Reply:
I was being myopic when I made the change. BTW the utility shopEditor it is broken. I will look at fixing it later.